components/02-elements/tab/tab.js

```diff
@@ -3,38 +3,31 @@
 (function() { // eslint-disable-line
   const tab                = [...document.querySelectorAll('.tab')],
         activeTitleClass   = 'tab__title--active',
-        activeContentClass = 'tab__content--active',
-        mediaQuery         = window.matchMedia('(min-width: 768px)');
```

Comment:
Why you removed all code related to MQ? I'm pretty sure it's required in Alpaca.

Reply:
It was added by me, b/c I wanted to add opened first tab on resize, but it's unecessary I think.